Makefile

```diff
@@ -1,8 +1,11 @@
+clean:
+	rm -rf docs/_build
+
 docs:
-	sphinx-build -W -a docs/reference/ docs/reference/_build/html
+	sphinx-build -W -a docs/ docs/_build/html
 
-tutorials:
-	sphinx-build -W -a docs/tutorials/ docs/tutorials/_build/html
+docsWithTutorials:
```

Comment:
doesn't this need to be in the .PHONY list too?  I am not mr. Makefile tho so I am happy to use this just as an edification opportunity :)

Reply:
Good catch.  Completed.